iocage_lib/ioc_start.py

```diff
@@ -1167,3 +1268,183 @@ def find_bridge_mtu(self, bridge):
         ).split()
 
         return membermtu[5]
+
+    def _command(self, command, *args, fib=None, jail=None):
```

Comment:
@skarekrow we already have an exec interface. Perhaps that should be used instead ? It can be modified/improved to better handle scenarios but duplicating doesn't seem like a very good idea. Thoughts ?

Reply:
Agreed, looking at this section it should be easily adapted to IOCExec. Though the DEBUG Running Command: stuff would need to be added as that's quite handy.

Reply:
I saw all the ifconfig commands in `ioc_start.py` run with a thin `checkoutput` wrapper, so I jumped off from there. I see `IOCExec` now that you've mentioned it, and I can look at reworking my wrapper into that with any needed changes.